### ISSUE TITLE
Zoom/detail multiplier

### ISSUE THREAD

**Issue by Explosion-Scratch** [organicmaps/organicmaps#8171]
⚠ Have you searched for similar, already existing issues?

**Is your feature request related to a problem? Please describe.**
To view some types of objects, such as Archeological Sites, one must zoom in significantly. This prevents me from viewing many such things that are further than 50ft from each other or so. This problem also affects many other things, e.g. shops aren't visible past a certain amount of zooming out.

**Describe the ideal solution**
I would like to see a setting that multiplies the detail of the map. By default it would be 1 (current behavior), but if it were set to 2, this would show the detail of 2x zoom at 1x zoom.

**Describe alternatives you have considered**
- If you're looking for something specific, (in my case archeological sites) I can search for "Archeological Site" and have them pop up, but this doesn't work when you're looking across categories or simply want to see a more detailed rendering.

**Additional context**
_N/A_

**Comment by Explosion-Scratch** [comment]
This is a solution for (though not a duplicate of): #2901, #3460, #1193, #6223, #702, #5363

**Comment by Misalf-git** [comment]
At least OsmAnd and Cruiser have this.
This also helps with slow devices by showing less details when reversing the "detail zoom".

**Comment by Explosion-Scratch** [comment]
> At least OsmAnd and Cruiser have this. This also helps with slow devices by showing less details when reversing the "detail zoom".

Yeah, it has many implications, especially if there was a way to change this specifically for certain feature types

**Comment by pastk** [comment]
- related to #https://github.com/organicmaps/organicmaps/issues/6330

**Comment by samcarter** [comment]
This might also help with the problem https://github.com/organicmaps/organicmaps/issues/7677

**Comment by krzemyczek** [comment]
yep, that would be perfect. My biggest issue with this is app is not being able to see where the forests are on country view, which is problematic when making bicycle route
